### PR TITLE
Prepare release 3.0.0

### PR DESCRIPTION
**Pull request by mruoss**
- [x] Change version in mix.exs
- [x] Update Changelog
- [x] Update README
- [x] Sneak in a usage livebook 
- [ ] Define date of release in CHANGELOG.md

After merge:

- [ ] Create a GIT Tag
- [ ] Create a release on Github
- [ ] `mix hex.publish`

### CODE REVIEW

**Comment by mruoss** [comment]
@jlgeering Want to merge, create the release on GH and run [`mix hex.publish`](https://hex.pm/docs/publish)?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [3.0.0] - TBD
+
 ### Changed
 
 - use `Enum.map_join/3` indead of `Enum.map/2` and `Enum.join/2` as it's more efficient according to credo recommendations
-- fix #87 - change the serialization of timestamps to use the canonical (iso8601) format, i.e. before: `2022-07-31 14:48:48.000000000 Z` and now: `"2022-07-31T14:48:48Z"`
+- Change the serialization of timestamps to use the canonical (iso8601) format, i.e. before: `2022-07-31 14:48:48.000000000 Z` and now: `"2022-07-31T14:48:48Z"` ([#87](https://github.com/ufirstgroup/ymlr/issues/87), [#90](https://github.com/ufirstgroup/ymlr/pull/90))
```
Comment (jlgeering) [inline CHANGELOG.md:14]: @mruoss shall we add a comment here that it is compatible with Elixir 1.10 again?

Comment (mruoss) [reply CHANGELOG.md:16]: I added it above. You like?

Comment (jlgeering) [reply CHANGELOG.md:16]: me likes

**Pull request merged**